app/models/subscription/request/ContactInformation.scala

```diff
@@ -173,15 +173,28 @@ object SecondaryContact {
   }
 
   def convertTo(userAnswers: UserAnswers): Either[ApiError, Option[SecondaryContact]] = {
+
+    val individualOrSoleTrader = {
+      (userAnswers.get(WhatAreYouRegisteringAsPage), userAnswers.get(BusinessTypePage)) match {
+        case (Some(RegistrationTypeIndividual), _) => true
+        case (_, Some(BusinessType.Sole))          => true
+        case _                                     => false
+      }
+    }
+
     val secondaryContactNumber = userAnswers.get(SndContactPhonePage)
 
-    (userAnswers.get(SecondContactPage), userAnswers.get(SndConHavePhonePage)) match {
-      case (Some(true), Some(true)) if secondaryContactNumber.isDefined =>
+    (individualOrSoleTrader, userAnswers.get(SecondContactPage), userAnswers.get(SndConHavePhonePage)) match {
+      case (true, _, _) if secondaryContactNumber.isDefined =>
```

Comment:
@iservport for individual/sole secondary contact question is not asked, this condition will not be invoked

better to remove 
```suggestion
      case (true, _, _) =>
```